test/integration/README.md

```diff
@@ -0,0 +1,86 @@
+The Envoy integration test framework is designed to make it simple to test downstream-Envoy-upstream
+communication.  In the common case, one
+
+- Sends a request from downstream through Envoy
+- Verifies the request is received upstream, and possibly inspects elements of the headers or body
+- Sends a response from upstream through Envoy
+- Verifies the request is received downstream, again possibly inspecting headers and body.
+
+For the simplest variant of this, one could do the following.
+
+```
```

Comment:
You can use GH C++ syntax highlighting here, I think something like: \`\`\`c++

Reply:
Oh that's way better, thanks